test/streaming/live-dialog.ts

```diff
@@ -0,0 +1,78 @@
+import { focusChild, focusMain, test, useSpectron } from '../helpers/spectron';
+import { getFormInput } from '../helpers/spectron/forms';
+import { sleep } from '../helpers/sleep';
+import { logIn } from '../helpers/spectron/user';
+
+useSpectron({ appArgs: '--nosync' });
+
+test('Shows optimized encoder for specific games', async t => {
+  const { app } = t.context;
+  await logIn(t);
+
+  await focusMain(t);
+  await app.client.waitForExist('button=Go Live');
+  await app.client.click('button=Go Live');
+  await focusChild(t);
+  await app.client.waitForExist('label=Title');
+
+  // This part also tests game search in some way
+  app.client.setValue('.multiselect__input', 'starcraft');
+  const sc2Selector = '.multiselect__element=StarCraft II';
+
+  await app.client.waitForExist(sc2Selector, 10000);
+  await app.client.click(sc2Selector);
+
+  // Unsupported games show "use optimized encoder settings" without the game
+  t.true(await app.client.isExisting('.profile-default'));
+
+  /*
+   * Overwatch is a supported game, so it should display the following label:
+   * "Use optimized encoder settings for Overwatch".
+   * Unfortunately, something seems to be messing up webdriver and it can't find
+   * 'label=Use optimized encoder settings for Overwatch'
+   * so we workaround by adding a CSS class for when it's using the default profile
+   */
+  const owSelector = '.multiselect__element=Overwatch';
+  await app.client.setValue('.multiselect__input', 'Overwa');
+  await app.client.waitForExist(owSelector, 10000);
+  await app.client.click(owSelector);
+  await sleep(1000);
```

Comment:
that may be flaky, use waitForVisible/waitForExist

Reply:
There doesn't seem to be a selector I can tap in (we're looking for input's value as our wait condition) so I did something similar to what FormMonkey seems to do with dropdowns.